src/components/NewTodoForm/NewTodoForm.js

```diff
@@ -0,0 +1,127 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import './NewTodoForm.css';
+
+export class NewTodoForm extends React.Component {
+  state={
+    username: 'Choose a user',
+    title: '',
+    errortitle: false,
+    errorusername: false,
+  }
+
+  handleChange = (event) => {
+    const { name, value } = event.target;
+    const re = /[^\w ]+/;
+
+    this.setState({
+      [name]: value.replace(re, ''),
+      [`error${name}`]: false,
+    });
+  }
+
+  handleSubmit = (event) => {
+    const { title, username } = this.state;
+    const { onAdd, users } = this.props;
+
+    event.preventDefault();
+
+    if (!title.trim()) {
```

Comment:
trim title before adding it to state

Reply:
if I use trim() before adding value to state, there is no possibility to print space before next word in a title.